guides/source/active_record_migrations.md

```diff
@@ -589,14 +718,15 @@ add_foreign_key :articles, :authors
 ```
 
 This [`add_foreign_key`][] call adds a new constraint to the `articles` table.
-The constraint guarantees that a row in the `authors` table exists where
-the `id` column matches the `articles.author_id`.
+The constraint guarantees that a row in the `authors` table exists where the
+`id` column matches the `articles.author_id`.
```

Comment:
The constraint guarantees that a row in the `authors` table exists where the
`id` column matches the `articles.author_id` to ensure all reviewers listed in the articles table are valid authors listed in the authors table.